_tags

```diff
@@ -1,4 +1,5 @@
+true: use_ocamlfind
```

Comment:
J'ai un warning pour cette ligne. A quoi te sert cette option ? J'ai ceci : 

```File "_tags", line 1, characters 6-19:                                                         
Warning: the tag "use_ocamlfind" is not used in any flag or dependency declaration, so it will ### Development
 have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild. 
ml to disable this warning. ```